lib/libs/email/content/email-components.tsx

```diff
@@ -231,9 +231,8 @@ const WithdrawRAI = ({
 }) => (
   <Section>
     <Heading as="h2">
-      The OneMAC Submission Portal received a request to withdraw the Formal RAI Response. You are
-      receiving this email notification as the Formal RAI for {id} was withdrawn by {submitterName}{" "}
-      {submitterEmail}.
+      {`The OneMAC Submission Portal received a request to withdraw the Formal RAI Response. You are
+      receiving this email notification as the Formal RAI for ${id} was withdrawn by ${submitterName} ${submitterEmail}.`}
```

Comment:
```suggestion
      The OneMAC Submission Portal received a request to withdraw the Formal RAI Response. You are
      receiving this email notification as the Formal RAI for {id} was withdrawn by {submitterName} {submitterEmail}.
```

Since we're using React, would it make more sense to do this?